src/lib/score/widgets/IntSlider.cpp

```diff
@@ -91,7 +101,28 @@ void IntSlider::mouseReleaseEvent(QMouseEvent* event)
 {
   sliderReleased();
 }
-
+void IntSlider::createPopup(QPoint pos)
+{
+  auto w = new QSpinBox();
+  w->setWindowFlag(Qt::Tool);
+  w->setWindowFlag(Qt::FramelessWindowHint);
+  w->setValue(m_value);
+  w->setRange(m_min, m_max);
+
+  QObject::connect(
+      w,
+      SignalUtils::QSpinBox_valueChanged_int(),
+      this,
+      [=](int v)
+      {
+        this->setValue(v);
+        sliderMoved(this->value());
+      });
+  w->show();
+  w->move(pos.x(), pos.y());
+  QTimer::singleShot(5, w, [w] { w->setFocus(); });
```

Comment:
since this is done in multiple places maybe this can be refactored in a generic `createPopup(auto& slider, QPoint pos)` function ?